clients/admin-ui/src/features/privacy-requests/buttons/ReprocessButton.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+  Button,
+  ButtonProps,
+  forwardRef,
+  ListItem,
+  UnorderedList,
+} from "@fidesui/react";
+import { useState } from "react";
+
+import { useAppDispatch, useAppSelector } from "~/app/hooks";
+
+import { useAlert, useAPIHelper } from "../../common/hooks";
+import {
+  selectRetryRequests,
+  setRetryRequests,
+  useBulkRetryMutation,
+  useRetryMutation,
+} from "../privacy-requests.slice";
+import { PrivacyRequest } from "../types";
+
+type ReprocessButtonProps = {
+  buttonProps?: ButtonProps;
+  subjectRequest?: PrivacyRequest;
+};
+
+const ReprocessButton = forwardRef(
+  ({ buttonProps, subjectRequest }: ReprocessButtonProps, ref) => {
+    const dispatch = useAppDispatch();
+    const [isReprocessing, setIsReprocessing] = useState(false);
+    const { handleError } = useAPIHelper();
+    const { errorAlert, successAlert } = useAlert();
+
+    const { errorRequests } = useAppSelector(selectRetryRequests);
+    const [bulkRetry] = useBulkRetryMutation();
+    const [retry] = useRetryMutation();
+
+    const handleBulkReprocessClick = () => {
+      setIsReprocessing(true);
+      bulkRetry(errorRequests!)
+        .unwrap()
```

Comment:
Is there a reason to use `unwrap` to get the raw result instead of using RTK's result interface? We have some [helpers](https://github.com/ethyca/fides/blob/5a485387d8af247ec6479e4115088cbbb8394d77/clients/admin-ui/src/features/common/helpers.ts#L21-L22) for common handling of that type.